include/threads.hpp

```diff
@@ -31,6 +31,12 @@ const unsigned kKopsRestartCountPort = 7000;
 // used by the cache system
 const unsigned kCacheUpdatePort = 7150;
 
+// used by the local store
```

Comment:
Let's hold off on merging this until you merge the other local store stuff?

Reply:
Same with the `LocalStoreThread` below.

Reply:
ok

Reply:
Bump.